bin/katana/src/cli/node.rs

```diff
@@ -210,6 +210,8 @@ impl NodeArgs {
     }
 
     fn rpc_config(&self) -> RpcConfig {
+        const MAX_PAGE_SIZE: u64 = 10_000;
+        let page_size = Some(self.server.page_size.min(MAX_PAGE_SIZE));
```

Comment:
```suggestion
```

considering we already set a default value in the cli, i think its fine to not have this upper bound. i think we shouldn't impose constraints like this and let the power user do whatever they want with it. im also aware that leaving it unbounded can hurt the performance for user that is unaware of its implications. 

also if user set the value > 10k, this minimization will happen in silent, and thus may not be obvious for the user.